metadata/src/test/java/org/apache/kafka/metadata/MockRandom.java

```diff
@@ -15,16 +15,26 @@
  * limitations under the License.
  */
 
-package org.apache.kafka.metadata;
+package org.apache.kafka.server.util;
```

Comment:
This package doesn't match the file path. Did you mean to move this file to a different project and package?